merkle_util_test.go

```diff
@@ -250,6 +251,33 @@ func TestProveWithdrawals(t *testing.T) {
 	assert.True(t, flag, "Historical Summary Block Root Proof %v failed")
 }
 
+func TestUnmarshalSSZVersionedBeaconState(t *testing.T) {
+	oracleStateBytes, err := oracleState.MarshalSSZ()
+	if err != nil {
+		fmt.Println("error", err)
+	}
+
+	versionedBeaconState, err := beacon.UnmarshalSSZVersionedBeaconState(oracleStateBytes)
+	if err != nil {
+		fmt.Println("error", err)
+	}
+	assert.Equal(t, versionedBeaconState.Version, spec.DataVersionDeneb, "Version %v failed")
+	assert.Nil(t, err, "Error %v failed")
+}
+
+func TestMarshalSSZVersionedBeaconState(t *testing.T) {
+	versionedBeaconState, err := beacon.CreateVersionedState(&oracleState)
+	if err != nil {
+		fmt.Println("error", err)
+	}
+
+	_, err = beacon.MarshalSSZVersionedBeaconState(versionedBeaconState)
+	if err != nil {
+		fmt.Println("error", err)
+	}
+	assert.Nil(t, err, "Error %v failed")
```

Comment:
can you marshal oracleState and make sure the bytes are equal

Reply:
done